### ISSUE TITLE
Dropdown panel is too big when using virtual scroll

### ISSUE THREAD

**Issue by Nosfit** [primefaces/primeng#9815]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
```
**Current behavior**
When virtual scroll is enabled and dropdown has few items, the panel is too big and has a lot of free space at the bottom. The same behavior when you use dropdown with filter and virtual scrolling, after filtering dropdown has 1 item and height 200px.

 
![Screenshot 2021-01-25 at 15 33 34](https://user-images.githubusercontent.com/12278612/105713201-81234b00-5f23-11eb-8be4-f47cfa10de6f.png)

**Expected behavior**
The panel should be big enough to fit available options.

**Minimal reproduction of the problem with instructions**
i use stackblitz example from primeng docs. just add virtual scrolling to all dropdowns. all of them works incorect, especially group one.
https://stackblitz.com/edit/primeng-dropdown-demo-virtualscroll-bug

**Comment by jcastellscan** [comment]
that's true, seems there is a fixed scrollHeight of 200px. Is somebody taking a look at it? I cannot use it as it is right now? height should be dynamic based on the elements available. scrollHeight might have to be changed to maxScrollheight

**Comment by ikupresanin** [comment]
Any updates?

**Comment by mazzalotti** [comment]
I have a possible solution to this problem. how can i contribute?

**Comment by EsraaMahmmoud77** [comment]
any updates about this issue?

**Comment by cetincakiroglu** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
